packages/neuron-ui/src/stories/Overview.stories.tsx

```diff
@@ -61,14 +61,11 @@ const states = {
   'Has more than 10 Activities': stateTemplate,
 }
 
-const OverviewWithRouteProps = (props: StateWithDispatch) => (
-  <Route path="/" render={routeProps => <Overview {...routeProps} {...props} />} />
-)
-
 const stories = storiesOf(`Overview`, module).addDecorator(StoryRouter())
 
 Object.entries(states).forEach(([title, props]) => {
-  stories.add(title, () => <OverviewWithRouteProps {...props} />)
+  console.info(props)
```

Comment:
Is this log expected ?

Reply:
Yes, it's used to print the expected state of the component in the storybook. Due to the `useContext` hook used in these components, the states cannot be observed directly.

Reply:
ok

Reply:
BTW, all the codes in `/stories` are just for developing and testing.